Content.Shared/Damage/Components/DamageableComponent.cs

```diff
@@ -47,6 +47,31 @@ public sealed partial class DamageableComponent : Component
         [DataField(readOnly: true)] //todo remove this readonly when implementing writing to damagespecifier
         public DamageSpecifier Damage = new();
 
+        /// <summary>
+        ///     The current damage used for things like state calculations
+        /// </summary>
+        /// <remarks>
+        ///     Only used on certain damage containers (E.G. People and mice, not vending machines)
+        /// </remarks>
+        [DataField]
+        public DamageSpecifier DamageEffective = new();
+
+        /// <summary>
+        ///     If whatever entity this component is contained is should experience softcrit
+        /// </summary>
+        [DataField, ViewVariables(VVAccess.ReadWrite)]
+        public bool SoftCritEligible = false;
+
+        /// <summary>
+        ///     The amount of time it takes for <see cref="DamageEffective"/> to catch up with <see cref="Damage"/>
+        ///     This value is decreased with respect to how much damage a biological container has taken,
+        ///     becoming 0 if it has died
+        ///     Ex: If this is 2 seconds, then a person that has instantly reduced to critical damage from no damage
+        ///     would take 1 second to crit, because crit is halfway between perfectly healthy and dead.
+        /// </summary>
+        [DataField]
+        public TimeSpan DamageLerpTimeZeroDamage = TimeSpan.FromSeconds(20);
+
```

Comment:
As mentioned on discord damageable should not be infected with this stuff, it should be kept separate.